### PR TITLE
Add resources section

### PR DESCRIPTION
**Pull request by ryanseddon**
This is an almost straight copy of the resources page we have now with a tiny cleanup and removal of yepnope references.

This will round out the last bit of this site.

screen shot 2015-05-24 at 4 15 07 pm

### CODE REVIEW

**Comment by ryanseddon** [comment]
@Modernizr/team-modernizr ping

File: frontend/templates/index.hbs [hunk 1]
```diff
@@ -24,34 +24,7 @@
...
-            <h3 class="box">Talk to us</h3>
-            <ul class="box">
-              <li><a href="https://twitter.com/modernizr">Twitter</a></li>
-              <li><a href="https://stackoverflow.com/questions/tagged/modernizr">StackOverflow</a></li>
-              <li><a href="https://github.com/modernizr/modernizr/issues">GitHub</a></li>
-              <li><a href="https://paulirish.wufoo.com/forms/p1hsxsru1ohiv0u/">Contact</a></li>
-            </ul>
-          </div>
-        </div>
+        {{> partials/boxes latestPosts=latestPosts}}
```
Comment (patrickkettner) [inline frontend/templates/index.hbs]: you are a smart man.

**Comment by patrickkettner** [comment]
(So I thought I posted this, but the wifi here is crappy so maybe it didnt go through - sorry if this is spam)

The "let us know" button seems a bit weird. I know its the call to action for the page, but its _so_ much more apparent then the light grey explanation text it feels disconnected. I would either move it to a pink inline link, or change the wording on the button a bit to be more apparent ("submit a resource!" or something)

**Comment by patrickkettner** [comment]
otherwise, +1 awesome work

**Comment by ryanseddon** [comment]
Button now says "Suggest a resource" and links out to a prefilled github issue rather than being lost in the twitter ether.

screen shot 2015-05-28 at 09 20 52

**Comment by patrickkettner** [comment]
"...but if you know of a great resource for this page." seems incomplete.

otherwise, +2

**Comment by ryanseddon** [comment]
How about:
> This section contains useful resources that we plan to keep up-to-date on all things Modernizr.

**Comment by patrickkettner** [comment]
boom

**Comment by patrickkettner** [comment]
if that is the only change, merge it in when done

**Comment by ryanseddon** [comment]
Yep

**Comment by patrickkettner** [comment]
:D!

**Comment by patrickkettner** [comment]
awesome work, my good man

**Comment by ryanseddon** [comment]
I'm just standing on your shoulders

**Comment by patrickkettner** [comment]
apropos of nothing, every single australian I've met at jsconf this week is just stoked that I get to work with "ryan effing seddon"

**Comment by ryanseddon** [comment]
My bribes worked!

**Comment by patrickkettner** [comment]
hahah

**Pull request merged**